modules/integration/pages/ledger-quick-start.adoc

```diff
@@ -131,7 +131,7 @@ Listening on 0.0.0.0:8080
 Starting Rosetta API server
 ....
 +
-By default, the software connects to the ledger canister running on the Internet Computer production network.
+By default, the software *does not* connect to the ledger canister running on the Internet Computer production network, but rather it connects to one of the testnets.
```

Comment:
the Internet Computer production network -> the Internet Computer blockchain mainnet